packages/contracts-bedrock/test/setup/Upgrade.s.sol

```diff
@@ -0,0 +1,193 @@
+        try vm.readFile("test/fixtures/standard-versions.toml") returns (string memory data_) {
+            _stdVersionsToml = data_;
+        } catch {
+            require(false, "Upgrade: cannot find standard-versions.toml");
+        }
+
+        vm.parseToml(_stdVersionsToml, ".releases[\"op-contracts/v1.6.0\"]");
+        // Save OPCM contracts
+        save(
+            "L1CrossDomainMessenger",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(".releases[\"op-contracts/v1.6.0\"].l1_cross_domain_messenger.implementation_address")
+            )
+        );
+        save(
+            "OptimismMintableERC20Factory",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(
+                    ".releases[\"op-contracts/v1.6.0\"].optimism_mintable_erc20_factory.implementation_address"
+                )
+            )
+        );
+        save(
+            "SystemConfig",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(".releases[\"op-contracts/v1.6.0\"].system_config.implementation_address")
+            )
+        );
+        save(
+            "L1StandardBridge",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(".releases[\"op-contracts/v1.6.0\"].l1_standard_bridge.implementation_address")
+            )
+        );
+        save(
+            "L1ERC721Bridge",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(".releases[\"op-contracts/v1.6.0\"].l1_erc721_bridge.implementation_address")
+            )
+        );
+        save(
+            "OptimismPortal2",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(".releases[\"op-contracts/v1.6.0\"].optimism_portal.implementation_address")
+            )
+        );
+        save(
+            "DisputeGameFactory",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(".releases[\"op-contracts/v1.6.0\"].dispute_game_factory.implementation_address")
+            )
+        );
+        save(
+            "DelayedWETHProxy",
+            vm.parseTomlAddress(
+                _stdVersionsToml,
+                string.concat(".releases[\"op-contracts/v1.6.0\"].delayed_weth.implementation_address")
+            )
+        );
+        save(
+            "PreimageOracle",
+            vm.parseTomlAddress(
+                _stdVersionsToml, string.concat(".releases[\"op-contracts/v1.6.0\"].preimage_oracle.address")
+            )
+        );
+        save(
+            "Mips",
+            vm.parseTomlAddress(_stdVersionsToml, string.concat(".releases[\"op-contracts/v1.6.0\"].mips.address"))
+        );
+
+        // need to get this from bootstrap output
+        // save("OPContractsManager", _json.readAddress("$.opChainDeployments[0].opContractsManager"));
```

Comment:
Does this help? https://github.com/ethereum-optimism/superchain-registry/blob/2b146f828682ef3784f76e39d15f5ad16e17c302/superchain/configs/mainnet/superchain.toml#L4

Alternatively, should we open a feature request to have this address in the `state.json`?